This test tries to access /var/lib/dbus/machine-id because of its usage of IBus.

diff --git a/testsuite/meson.build b/testsuite/meson.build
index f842a07..bd31ec8 100644
--- a/testsuite/meson.build
+++ b/testsuite/meson.build
@@ -34,7 +34,6 @@ testsuite_sources = [
   ['test-styleschememanager'],
   ['test-undo-manager'],
   ['test-utils'],
-  ['test-view'],
 ]
 
 foreach test: testsuite_sources
